packages/edit-widgets/src/index.js

```diff
@@ -23,24 +23,32 @@ import { create as createLegacyWidget } from './blocks/legacy-widget';
 import * as widgetArea from './blocks/widget-area';
 import Layout from './components/layout';
 
+let hasInitialized = false;
```

Comment:
we should add some comment about what is this here for.

Reply:
Hmm, I'm not sure what to add here. As the code itself is pretty self-explanatory 🤔? I most likely have a blind spot here though.